spec/06-expressions.md

```diff
@@ -329,8 +329,8 @@ arguments, the following conditions must hold.
 - The names $x_i$ of all named arguments are pairwise distinct and no named
   argument defines a parameter which is already specified by a
   positional argument.
-- Every formal parameter $p_j:T_j$ which is not specified by either a positional
```

Comment:
Maybe (I'm not sure) it's wrong to talk about implicit arguments here: the spec here speaks about an application, which means `f(args)`. In this case, no implicits are inserted by the compiler.

So your clarification could be a note below the list of conditions.